Modules/Loadable/SubjectHierarchy/Widgets/qMRMLSubjectHierarchyTreeView.cxx

```diff
@@ -1882,9 +1883,9 @@ void qMRMLSubjectHierarchyTreeView::applyReferenceHighlightForItems(QList<vtkIdT
       if (referencedItem && referencedItem != itemID)
         {
         QStandardItem* item = sceneModel->itemFromSubjectHierarchyItem(referencedItem, nameColumn);
-        if (item && !d->HighlightedItems.contains(referencedItem))
+        if (item && !d->HighlightedItems.contains(referencedItem) && this->sortFilterProxyModel()->filterAcceptsItem(referencedItem))
           {
-          item->setBackground(d->IndirectReferenceColor);
+          item->setBackground(QColor::fromRgb(255, 255, 170));
```

Comment:
Should these attribute be removed as well:

https://github.com/Slicer/Slicer/blob/231fc4f35f63606fa81ffe53b330efdc2477159b/Modules/Loadable/SubjectHierarchy/Widgets/qMRMLSubjectHierarchyTreeView.cxx#L145-L148

Reply:
No, sorry, this is a merge mistake due to a very recent change in the same file and the same function. I actually need to add those instead of the colors. I'll fix this asap.